package.json

```diff
@@ -7,55 +7,55 @@
     "url": "https://github.com/reactjs/reactjs.org"
   },
   "dependencies": {
-    "babel-eslint": "^8.0.1",
-    "eslint": "^4.8.0",
-    "eslint-config-fbjs": "^2.0.0",
+    "babel-eslint": "^8.2.6",
+    "eslint": "^4.19.1",
+    "eslint-config-fbjs": "^2.1.0",
     "eslint-config-react": "^1.1.7",
     "eslint-plugin-babel": "^4.1.2",
-    "eslint-plugin-flowtype": "^2.39.1",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-flowtype": "^2.50.3",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^2.7.0",
+    "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-relay": "^0.0.19",
     "flow-bin": "^0.56.0",
```

Comment:
Bump flow to at least 0.75 to get the LSP protocol and all the nice autocompletion that comes with that

Reply:
Bumping it to latest caused some errors so that's why I reverted it

Reply:
Flow changed a lot since then, it works if you update some suff in the `.flowconfig`.

Comment out or remove this
```
unsafe.enable_getters_and_setters=true
```

And update the flow version at the bottom

```
[version]
^0.121.0
```

Then reload the vscode plugin and it should work.

Side note, it would be really cool if you could comment on other files in a PR, can you do it and I just don't know about it??